dev/com.ibm.ws.anno/src/com/ibm/ws/annocache/targets/cache/internal/TargetCacheImpl_ReaderBinary.java

```diff
@@ -466,7 +480,8 @@ public void readEntire(TargetsTableContainersImpl containerTable) throws IOExcep
     }
 
     public void readFragment(TargetsTableContainersImpl containerTable) throws IOException {
-        requireContainersHeader();
+        String readVersion = requireContainersHeader();
+        boolean readStamps = readVersion.equals(TargetCache_BinaryConstants.CONTAINER_TABLE_VERSION_20);
```

Comment:
In the non Binary class you did greater than or equal to.  Should that be done here as well?

Reply:
Fixing.